.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,16 @@
-\<your comments for this PR go here\>
```

Comment:
I think we generally want a link to the ticket in the PR description, so it might be worth adding that here as well!